html/selionGrid.html

```diff
@@ -34,39 +34,28 @@ <h2 id="prerequisites">Prerequisites</h2>
   <h2 id="start-selion-grid">Start the Grid Hub</h2>
   <p>The grid hub serves as the central point. It will receive all test case requests from all
       clients and distribute them to the nodes registered with the grid.</p>
-  <p>Open the command prompt and navigate to the location of the grid binary. Type in the following command:</p>
-  <p>Windows</p>
+  <p>Open the command prompt and navigate to the location of the grid jar file. Type in the following command:</p>
 
-  <pre><code class="language-sh">autobot_start_Grid.bat</code></pre>
-
-  <p>Linux / Mac</p>
-
-  <pre><code class="language-sh">sh autobot_start_Grid.sh</code></pre>
+  <pre><code class="language-sh">java -jar SeLion-Grid.jar -role hub</code></pre>
 
   <p>
       The grid hub will start up using the port <var>4444</var> by default. You can view it by connecting
-      to <kbd>http://localhost:4444</kbd> with your preferred browswer.
+      to <kbd>http://localhost:4444</kbd> with your preferred browser. By default all the required files will be downloaded or 
+      extracted to &lt;user.home&gt;/.selion directory. To specify the different directory configure system property
+      <code>-DselionHome=&lt;folderPath&gt;</code> to the java process.
   </p>
 
   <h2 id="start-selion-node">Start the Grid Node</h2>
-  <p>The grid node will execute the test cases. Edit the file <code>autobot_start_Node.bat</code> or
-      <code>autobot_start_Node.sh</code> to update the <var>HUB_URL</var> which represents SeLion Grid URL. Update the entry to use
-      the local grid as below:</p>
-  <pre><code class="language-sh">set HUB_URL=http://localhost:4444/grid/register</code></pre>
-  <p>Open the command prompt and navigate to the Enhanced Grid folder. Execute the below command to
+  <p>The grid node will execute the test cases.</p>
+  <p>Open the command prompt and navigate to the location of the grid jar file. Execute the below command to
       start the node</p>
-  <p>Windows</p>
-
-  <pre><code class="language-sh">autobot_start_Node.bat</code></pre>
-
-  <p>Linux / Mac</p>
 
-  <pre><code class="language-sh">sh autobot_start_Node.sh</code></pre>
+  <pre><code class="language-sh">java -jar SeLion-Grid.jar -role node -hub http://localhost:4444/grid/register</code></pre>
```

Comment:
You shouldn't need to specify `-hub` if you are attaching to a hub running on the same machine but it certainly doesn't hurt.